source_me.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
```

Comment:
can we rename this file to build_env.sh ?

Reply:
Done